docs/snippets/vue/app-story-with-mock.vue3.js.mdx

```diff
@@ -0,0 +1,37 @@
+```js
+// App.stories.js
+
+import App from './App.vue';
+
+export default {
+  title: 'App',
+  component: App
+};
+
+const Template = (args) => ({
+  components: { App },
+  setup() {
+    return { args };
+  }
+  template: "<App />"
```

Comment:
```suggestion
  template: '<App v-bind="args" />'
```

We always need to `v-bind="args"` in templates to "spread" the story args as props.

Reply:
@phated fixed with the latest commit. Thanks for the clarification.